Config.cmake.in

```diff
@@ -0,0 +1,20 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+
+include(CMakeFindDependencyMacro)
+
+# we do not add the other dependencies because these are header files lib
+if(${CMAKE_VERSION} VERSION_LESS "3.15.0")
```

Comment:
Same question as on other packages: Is this distinction needed?

Reply:
Yes otherwise the dependencies are not seen by child packages.

Reply:
I see.  All good then :+1: